### DISCUSSION TITLE
Have a setting to disable the epilepsy warning.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#20249] [Ideas]
A setting to disable the epilepsy warning would be nice, as I believe it's a bit too long. It'd function similarly to the setting to disable storyboard/video.

(There was a small discussion about this setting in #20290. Overall consensus seems to be to show the warning in the current warning screen.)

**Comment by peppy** [comment]
I think adding a setting may be okay, but it should only be shown in the main settings overlay (not the player loading screen settings).

My only apprehension about adding a toggle is that someone may toggle this on a shared PC and it wouldn't display when it actually would be of help. But honestly, the epilepsy warning is heavily misused by mappers currently. At some point in the future I'll automate this based off known criteria, at which point a toggle may be less important (since it will show less often).
